kymatio/scattering1d/frontend/tensorflow_frontend.py

```diff
@@ -3,17 +3,20 @@
 
 from ...frontend.tensorflow_frontend import ScatteringTensorFlow
 from ..core.scattering1d import scattering1d
+from ..core.timefrequency_scattering import timefrequency_scattering
 from ..utils import precompute_size_scattering
-from .base_frontend import ScatteringBase1D
+from .base_frontend import (ScatteringBase1D, TimeFrequencyScatteringBase1D,
+                            _check_runtime_args_jtfs)
 
 
 class ScatteringTensorFlow1D(ScatteringTensorFlow, ScatteringBase1D):
-    def __init__(self, J, shape, Q=1, max_order=2, average=True,
-            oversampling=0, vectorize=True, out_type='array', backend='tensorflow',
-                 name='Scattering1D'):
+    def __init__(self, J, shape, Q=1, T=None, max_order=2, average=True,
+            oversampling=0, vectorize=True, out_type='array', pad_mode='reflect',
```

Comment:
why do there need to be modifications to this part of the codebase in this PR? I'd prefer to focus only on jtfs. I am hoping this is just an artefact of this PR being stacked on top of others.

Reply:
I don't follow. JTFS has its own functionality and own kwargs to control it.